Documentation/alm/prometheus-ocs.md

```diff
@@ -0,0 +1,62 @@
+# Prometheus Open Cloud Service
+
+Tectonic’s Prometheus Open Cloud Service provides a one-click, fully managed, application monitoring and metrics stack for both operations and applications teams on-top of a Tectonic cluster. Use Prometheus’s Alertmanager to route, email, page, or message teams when something goes wrong with an application or the container infrastructure. Prometheus OCS is:
+
+* **Highly available:** Configure redundancy to ensure important metrics are never missed in production or save resources in development environments by running a single instance.
+* **Kubernetes Native:** Use native Kubernetes paradigms like Pod label selectors and automatically track all application containers even as they are upgraded, destroyed, or rescheduled.
+* **High performance:** Containers encourage the creation of more services and CoreOS has ensured Prometheus can track all of those services while using minimal resources. (For more information on Prometheus, see the blog post [Prometheus, the backbone of container monitoring, hits 2.0][prom-20].)
+
+Allows you to deploy and manage Prometheus instance into any namespace.
+
+## Deploying Prometheus OCS
+
+Use Tectonic Console to enable the Prometheus OCS for selected namespaces.
+
+By default, objects created using the Prometheus OCS will be labeled `prometheus=k8s`.
+
+Using the Prometheus Open Cloud Service to deploy a Prometheus instance will create the following Kubernetes objects:
+* A Prometheus CRD
+* A Prometheus Service Monitor
+* A Prometheus Stateful Set
+* A Prometheus Secret
+
+## Ingesting metrics
+
+Once enabled, go to *https://<your-cluster-dns>/prometheus* to access Prometheus monitoring for the cluster. Select *Status > Targets* to confirm that Prometheus is correctly configured and ingesting metrics.
```

Comment:
Ingress is not automatically setup, as there are too many ways to configure it. This is good, just make sure you mention that one needs to setup some way of accessing the UI first.